packages/forms/src/UIForm/fields/TextArea/TextArea.component.js

```diff
@@ -53,7 +53,7 @@ if (process.env.NODE_ENV !== 'production') {
 			readOnly: PropTypes.bool,
 			rows: PropTypes.number,
 			title: PropTypes.string,
-		}),
+		}).isRequired,
```

Comment:
schema is required and none of its component is, isn't it more safe then to rely on a default props set to an empty literral object ?